live-ingester/pom.xml

```diff
@@ -93,6 +93,12 @@
             <artifactId>lombok</artifactId>
             <scope>provided</scope>
         </dependency>
+        <dependency>
+            <groupId>org.apache.pdfbox</groupId>
+            <artifactId>pdfbox</artifactId>
+            <version>2.0.26</version>
+            <scope>test</scope>
+        </dependency>
```

Comment:
is this dependency necessary?

Reply:
This was left unintentionally. Thanks for catching it !
Deleted in [22a68e1](https://github.com/Alfresco/hxinsight-connector/pull/142/commits/22a68e10c384f2f12b28bf6864bc9963fd6b39f1)